packages/eslint-config-4catalyzer/rules.js

```diff
@@ -49,4 +49,5 @@ module.exports = {
   // We very seldom intentionally use async functions without await.
   'require-await': 'error',
   'import/prefer-default-export': 'off',
+  'lines-between-class-members': 'off',
```

Comment:
i like this though ):

Reply:
oh does `--fix` not handle that?

Reply:
maybe it does...the docs say it does, but i had to do most of them by hand for react-bootstrap

Reply:
\\: i like this rule though

Reply:
fiiiiiine